source/RouteViewer/Audio/Sounds.Update.cs

```diff
@@ -23,7 +24,14 @@ protected override void UpdateLinearModel(double timeElapsed)
 			 * */
 			Vector3 listenerPosition = Camera.AbsolutePosition;
 			Orientation3 listenerOrientation = new Orientation3(Camera.AbsoluteSide, Camera.AbsoluteUp, Camera.AbsoluteDirection);
-			Vector3 listenerVelocity = Camera.AlignmentSpeed.Position;
+			Vector3 listenerVelocity;
```

Comment:
This change should be undone.
I think RouteManager TrainManager.PlayerTrain is null.

Reply:
Will double check, but if I recall it's not actually called.

Leaving it there is ?possibly? marginally better, as it means the code is common between Route Viewer and the main sim.

Reply:
Correctly TrainManager.PlayerTrain.Cars is null.
Try launching RouteViewer.
System.NullReferenceException occurs.

Reply:
Almost right- It did crash, but not with this one....

I've one two things to fix this, which is I think a little better-
* Added a check for whether we have a loaded route before running the update functions for sounds etc.
* Changed the current camera view to be track based, not interior.

Reply:
Thanks! I've confirmed.